public/app/features/admin/migrate-to-cloud/EmptyState/InfoPane1.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { Box, Stack } from '@grafana/ui';
+import { t, Trans } from 'app/core/internationalization';
+
+import { InfoItem } from './InfoItem';
+
+export const InfoPane1 = () => {
+  return (
+    <Box padding={2} backgroundColor="secondary">
+      <Stack gap={2} direction="column">
```

Comment:
I think we'd rather add the `direction` prop to the Box and then we'd get rid of the extra Stack here. I think the only reason we haven't added `direction` prop to `Box` is because we haven't had any need for it yet.